rpm-ostree-container-bootc.ks.in

```diff
@@ -0,0 +1,42 @@
+#test name: rpm-ostree-container-bootc
+# for bootc/bootupd, remote and stateroot ostreecontainer options
+# depends on the referenced ostree container being bootable
+
+# Use the default settings.
+%ksappend common/common_no_payload.ks
+
+# The RPM OSTree source is set up by the test script
+
+# Reboot the installed system.
+reboot
+
+# Validate on the first boot.
+%ksappend validation/success_on_first_boot.ks
+
+%post
+# Checks in %post
+# Check of the remote URL is appended by the test script
+
+# Checks after boot
+cat >> /opt/kickstart-tests/kickstart-test.sh << 'EOF'
```

Comment:
This part was failing for me with file not found, updated here: https://github.com/rhinstaller/kickstart-tests/pull/1116/commits/41045cf2ce7167c6d2b5581816eed4a655b35c59